packages/mui-codemod/src/deprecations/all/deprecations-all.js

```diff
@@ -39,6 +40,9 @@ export default function deprecationsAll(file, api, options) {
   file.source = transformButtonGroupClasses(file, api, options);
   file.source = transformChipClasses(file, api, options);
   file.source = transformCircularProgressClasses(file, api, options);
+  file.source = transformSpeedDialProps(file, api, options);
+  file.source = transformTableSortLabelClasses(file, api, options);
+  file.source = transformTextFieldProps(file, api, options);
```

Comment:
These should be sorted alphabetically, also, `transformSpeedDialProps` and `transformTextFieldProps` are already called below.

Reply:
Got it.. i think this happened while fixing merge conflicts. anyway i fixed here https://github.com/mui/material-ui/pull/42281/commits/dc0d43a9e814117b67e91bda7ee950cae27c84b9